### PR TITLE
♻️ refactor stop service command ⚠️🚨

### PR DESCRIPTION
**Pull request by GitHK**
## ⚠️ Dev-ops
This code has to be deployed with all dynamic-services stopped. The service stop API from `webserver`-> `dynamic-scheduler` changed in a backwards incompatible way.

## What do these changes do?

Adds `project_id` and `user_id` to the stop command sent to the `dynamic-scheduler`. These changes are required to cover an edge case: enables stopping a service that is not tracked by the scheduler.

These changes are part of https://github.com/ITISFoundation/osparc-simcore/pull/5892

## Related issue/s

- required by https://github.com/ITISFoundation/osparc-simcore/pull/5892

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py [hunk 1]
```diff
@@ -60,17 +61,13 @@ async def run_dynamic_service(
 async def stop_dynamic_service(
     rabbitmq_rpc_client: RabbitMQRPCClient,
     *,
-    node_id: NodeID,
-    simcore_user_agent: str,
-    save_state: bool,
+    rpc_dynamic_service_stop: RPCDynamicServiceStop,
```
Comment (sanderegg) [inline packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py:64]: Suggestion: I don't think we care about the technology you use for stopping services. So I would suggest to remove RPC from the names here.

Comment (sanderegg) [reply packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py:64]: and actually same for creation

**Review by sanderegg — APPROVED** [review]
not sure I get why this change is needed. but I guess it will come with the next PR..

File: packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py (hunk 1 above)
Comment (GitHK) [reply packages/service-library/src/servicelib/rabbitmq/rpc_interfaces/dynamic_scheduler/services.py]: renamed both.

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks! :100:

**Pull request merged**